src/cartridge/include/cartridge/utils.h

```diff
@@ -0,0 +1,297 @@
+    {0x92, "Video system"},
+    {0x96, "Yonezawa/s'pal"},
+    {0xA4, "Konami (Yu-Gi-Oh!)"},
+    {0x01, "Nintendo R&D1"},
+    {0x18, "Hudson Soft"},
+    {0x22, "pow"},
+    {0x28, "Kemco Japan"},
+    {0x31, "Nintendo"},
+    {0x34, "Konami"},
+    {0x38, "Hudson"},
+    {0x42, "Atlus"},
+    {0x47, "Bullet-Proof"},
+    {0x51, "Acclaim"},
+    {0x54, "Konami"},
+    {0x57, "Matchbox"},
+    {0x60, "Titus"},
+    {0x67, "Ocean"},
+    {0x71, "Interplay"},
+    {0x75, "sci"},
+    {0x80, "misawa"},
+    {0x87, "tsukuda ori*"},
+    {0x93, "Ocean/Acclaim"},
+    {0x97, "Kaneko"},
+    {0x08, "Capcom"},
+    {0x19, "b-ai"},
+    {0x24, "PCM Complete"},
+    {0x29, "seta"},
+    {0x32, "Bandai"},
+    {0x35, "Hector"},
+    {0x39, "Banpresto"},
+    {0x44, "Malibu"},
+    {0x49, "irem"},
+    {0x52, "Activision"},
+    {0x55, "Hi tech entertainment"},
+    {0x58, "Mattel"},
+    {0x61, "Virgin"},
+    {0x69, "Electronic Arts"},
+    {0x72, "Broderbund"},
+    {0x78, "THQ"},
+    {0x83, "lozc"},
+    {0x91, "Chunsoft"},
+    {0x95, "Varie"},
+    {0x99, "Pack in soft"}};
+
+const std::map<uint8_t, std::string> sgb_flag_parse{
+    {0x00, "No SGB Support"}, {0x03, "Game Supports SGB functions"}};
+
+const std::map<uint8_t, std::string> cartridge_type_parse{
+    {0x00, "ROM ONLY"},
+    {0x19, "MBC5"},
+    {0x01, "MBC1"},
+    {0x1A, "MBC5+RAM"},
+    {0x02, "MBC1+RAM"},
+    {0x1B, "MBC5+RAM+BATTERY"},
+    {0x03, "MBC1+RAM+BATTERY"},
+    {0x1C, "MBC5+RUMBLE"},
+    {0x05, "MBC2"},
+    {0x1D, "MBC5+RUMBLE+RAM"},
+    {0x06, "MBC2+BATTERY"},
+    {0x1E, "MBC5+RUMBLE+RAM+BATTERY"},
+    {0x08, "ROM+RAM"},
+    {0x20, "MBC6"},
+    {0x09, "ROM+RAM+BATTERY"},
+    {0x22, "MBC7+SENSOR+RUMBLE+RAM+BATTERY"},
+    {0x0B, "MMM01"},
+    {0x0C, "MMM01+RAM"},
+    {0x0D, "MMM01+RAM+BATTERY"},
+    {0x0F, "MBC3+TIMER+BATTERY"},
+    {0x10, "MBC3+TIMER+RAM+BATTERY"},
+    {0xFC, "POCKET CAMERA"},
+    {0x11, "MBC3"},
+    {0xFD, "BANDAI TAMA5"},
+    {0x12, "MBC3+RAM"},
+    {0xFE, "HuC3"},
+    {0x13, "MBC3+RAM+BATTERY"},
+    {0xFF, "HuC1+RAM+BATTERY"}};
+
+const std::map<uint8_t, std::string> rom_size_parse{
+    {0x00, "32KB (no ROM banking)"},
+    {0x01, "64KB (4 banks)"},
+    {0x02, "128KB (8 banks)"},
+    {0x03, "256KB (16 banks)"},
+    {0x04, "512KB (32 banks)"},
+    {0x05, "1MB (64 banks)  - only 63 banks used by MBC1"},
+    {0x06, "2MB (128 banks) - only 125 banks used by MBC1"},
+    {0x07, "4MB (256 banks)"},
+    {0x08, "8MB (512 banks)"},
+    {0x52, "1.1MB (72 banks)"},
+    {0x53, "1.2MB (80 banks)"},
+    {0x54, "1.5MB (96 banks)"}};
+
+const std::map<uint8_t, std::string> cartridge_ram_parse{
+    {0x00, "None"},
+    {0x01, "2 KBytes"},
+    {0x02, "8 Kbytes"},
+    {0x03, "32 KBytes (4 banks of 8KBytes each)"},
+    {0x04, "128 KBytes (16 banks of 8KBytes each)"},
+    {0x05, "64 KBytes (8 banks of 8KBytes each)"}};
+
+const std::map<uint8_t, std::string> dest_code_parse{
+    {0x00, "Japanese"}, {0x01, "Non-Japanese"}};
```

Comment:
All files should end with a newline. You have this issue in a couple of other files as well, but clangformat should fix this for you.